"No tests found" now results in a failure.

dougthor42 opened the pull request:
Fixes #123.

This changes the return value and text coloring when no tests are found.

I wasn't able to figure out where any unit tests for this should go (I have limited time today), so if you could let me know where they should go, I'll try writing some tomorrow.

CleanCut commented:
Tests for `green/cmdline.py` go in `green/test/test_cmdline.py`.

Tests for `green/result.py` go in `green/test/test_result.py`

dougthor42 commented:
Alrighty, I updated things and added (well, modified) the relevant tests. I moved the logic for pass/fail to the `wasSuccessful()` method so that things work everywhere.

I also squashed the commits so that you have a nice clean history (though there were only 2 commits originally...).

Once tests pass, I think it's ready to go. They passed locally so hopefully they do the same on CI.
